app/frontend/src/search/ui/alert.js

```diff
@@ -0,0 +1,43 @@
+import { extractQueryParams } from '../../lib/utils';
+import '../../polyfill/remove.polyfill';
+
+const JOB_ALERT_URL = '/subscriptions/new';
+
+export const getJobAlertLink = url => {
```

Comment:
Just a note that some aspect of this isn't always working as intended https://dfedigital.atlassian.net/browse/TEVA-882

Reply:
sure i am aware of this ticket, i dont want to neccessarily try fixing anything already known like this as part of this PR